pkg/tempo/config.go

```diff
@@ -276,24 +299,50 @@ func (c *InstanceConfig) otelConfig() (*configmodels.Config, error) {
 		processorNames = append(processorNames, "batch")
 	}
 
-	otelMapStructure["processors"] = processors
+	if c.SpanMetrics != nil {
+		// Configure the metrics exporter.
+		exporters[c.SpanMetrics.MetricsExporter.Name] = c.SpanMetrics.MetricsExporter.Config
```

Comment:
> On second thought, would it make more sense to configure a prometheus exporter by default? The main use case will be to have the agent self-scrape the exported prometheus metrics and have them sent to the configured `remote_write`. I think it makes it a simpler feature for the users to enable (i.e. less configuration).
> 
> The future goal will be to hook this metrics directly to the agent's remote write, instead of having to scrape them locally.

Reply:
We need to define the exporter configuration first. If we just want to support Prometheus exporter then having a default namespace is fine.